model/database/spans.yaml

```diff
@@ -737,3 +737,9 @@ groups:
       - ref: db.cosmosdb.consistency_level
         requirement_level:
           conditionally_required: If available.
+      - ref: db.cosmosdb.regions_contacted
+        note: >
+            The `db.cosmosdb.regions_contacted` attribute is populated with all regions that were contacted during the operation.
+            If there is more than one region listed, it indicates that the operation was performed on multiple regions i.e. cross-regional call.
```

Comment:
is it need to override the note here, or is it ok to inherit the note from the registry?